test/apps/iodemo/ucx_wrapper.cc

```diff
@@ -471,9 +510,10 @@ void UcxContext::destroy_worker()
 
 unsigned UcxConnection::_num_instances = 0;
 
-UcxConnection::UcxConnection(UcxContext &context, uint32_t conn_id) :
-    _context(context), _conn_id(conn_id), _remote_conn_id(0),
-    _ep(NULL), _close_request(NULL), _ucx_status(UCS_OK)
+UcxConnection::UcxConnection(UcxContext &context) :
+    _context(context), _establish_cb(NULL),
+    _conn_id(UcxContext::get_next_conn_id()), _remote_conn_id(0), _ep(NULL),
+    _close_request(NULL), _ucx_status(UCS_INPROGRESS)
```

Comment:
this code section looks different in master branch

Reply:
seems like I applied clang formatting there